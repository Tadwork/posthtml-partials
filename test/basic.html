<mixin name="say" from>
	<p>Hello from {{ from }}!</p>
</mixin>

<div>
	<mixin name="say" from="me"></mixin>
</div>
