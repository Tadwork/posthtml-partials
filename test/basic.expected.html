

<div>
	
	<p>Hello from me!</p>

</div>
